client/app/(main)/storage/[id]/page.tsx

```diff
@@ -9,8 +9,11 @@ interface Props {
 }
 
 export default async function page({ params: { id: categoryId } }: Props) {
-  const category = await getCategoryById(categoryId)
-  const todolistsByDate = await getTodolistByDates(categoryId)
+  const categoryResponse = await getCategoryById(categoryId)
```

Comment:
Is it correct that I get a response when I assign a response to a destructuring assignment?